### DISCUSSION TITLE
Let's take advantage of Github functions such as the new "Issue & PR templates"!

### DISCUSSION THREAD

**Discussion by simbus82** [joomla/joomla-cms#37946] [General]
It's time ti convert all issues request form to the new github issue forms, they are far superior, accessible and they really help to manage all issues with an awesome labeling system.

See some examples:
- https://github.com/PrestaShop/PrestaShop/issues/new?assignees=&labels=Bug%2CNew&template=1_bug_report.yml
- https://github.com/home-assistant/core/issues/new?assignees=&labels=&template=bug_report.yml

(and see how good are user labels here https://github.com/PrestaShop/PrestaShop/issues)

**Comment by brianteeman** [comment]
it is pretty but having just looked at a quick sample of home-assitsant issues it didnt do anything to encourage people to complete the information and now the issue is clogged up with empty information (my 2c)

![image](https://user-images.githubusercontent.com/1296369/171232747-18ce259d-1ef7-41f5-866a-ed3f6eb13d38.png)
